layers/+tools/apache/packages.el

```diff
@@ -0,0 +1,27 @@
+;;; packages.el --- nginx layer packages file for Spacemacs.
```

Comment:
nginx -> apache

Reply:
I've made these fixes in a new commit.  I wasn't sure if you'd want a squashed commit.  If so let me know.